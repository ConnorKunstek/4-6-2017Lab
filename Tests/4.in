SDSSSSDDDDDSSSSSSSSSSSSSSSSSSS
DDDDDDDDDSSSSSSSSSSSSSSSSSSSSS
DSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
DDDDDDDDDDDSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSDSDFSSSSSSSSSSSSSSS
SSSSSSSSSSSSDDSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSDSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSDSSSSSSSSSSSSSSSSSSSS
SSSSSSSSSDDSSSSSSSSSSSSSSSSSSS
2 2
20 11
20 30
1 2
