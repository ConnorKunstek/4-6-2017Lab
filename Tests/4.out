Yes
Yes
No
Yes
Time: 0.006668 seconds

